src/js/parsers/captions/vttparser.js

```diff
@@ -471,6 +472,7 @@ VTTParser.prototype = {
         } catch (e) {
             throw e;
         }
+        /* eslint-enable no-useless-catch */
```

Comment:
To be fair, why bother catching? 🤣

Reply:
I honestly didn't want to change the code here as we might end up pulling this file in from `vtt.js` at some point. Might as well fix it though.